itl/plugins-contrib.d/network-components.conf

```diff
@@ -833,3 +833,98 @@ object CheckCommand "printer_health" {
 	vars.printer_health_hostname = "$check_address$"
 	vars.printer_health_mode = "supplies-status"
 }
+
+object CheckCommand "fiberchannel-brocade" {
+	import "ipv4-or-ipv6"
+	command = [ PluginContribDir + "/check_snmp_brocade" ]
+	arguments = {
+		"--hostname" = {
+			value = "$fiberchannel_brocade_hostname$"
+			description = "Hostname or IP-address of the server (SNMP mode only)"
+			required = true
+		}
+		"--community" = {
+			value = "$fiberchannel_brocade_community$"
+			description = "SNMP community of the switch (SNMP v1/2 only)"
+		}
+		"--snmpversion" = {
+			value = "$fiberchannel_brocade_snmpversion$"
+			description = "The SNMP protocol to use (default: 2c, other possibilities: 1,3)"
+		}
+		"--seclevel" = {
+			value = "$fiberchannel_brocade_seclevel$"
+			description = "The SNMP security level to use (default: noAuthNoPriv, othet possibilities: authNoPriv,authPriv)"
+		}
+		"--port" = {
+			value = "$fiberchannel_brocade_port$"
+			description = "The SNMP port to use (default: 161)"
+		}
+		"--context" = {
+			value = "$fiberchannel_brocade_context$"
+			description = "SNMPv3 context name"
+		}
+		# It seems --fc-port parameter is not working using its short form
+		"-P" = {
```

Comment:
If "--fc-port parameter is **not working** using its short form", why do you use the short form?